tests/test_rocrate_writer.py

```diff
@@ -262,6 +298,10 @@ def test_zip_crate(tmpdir, data_dir, builder, test_person_name, ro_crate_helpers
         entites = ro_crate_helpers.read_json_entities(Path(metadata_path).parent)
         ro_crate_helpers.check_crate(entites)
         validate_zip.close()
+    # external_manifests is true so check external manifests have been created
+    manifest_dir = tmpdir / "zipped_crate_manifests/"
+    assert manifest_dir.is_dir()
+    assert Path(manifest_dir / "ro-crate-metadata.json").is_file()
```

Comment:
I guess the `Path()` call isn't necessary, as the output of the `/` operator returns a `Path` anyway